app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -681,6 +683,35 @@ class PinPasswordActivityTest {
     }
   }
 
+  @Test
+  fun testPinPasswordActivity_checkKeyboardType_clickShowHidePassword_keyboardTypeIsSame() {
```

Comment:
Can you confirm that this fails without your fix in place?

Reply:
Yes

Reply:
Resolving on behalf of Ben